cmd/desync/untar.go

```diff
@@ -66,7 +66,10 @@ func runUntar(ctx context.Context, opt untarOptions, args []string) error {
 			return err
 		}
 		defer f.Close()
-		return desync.UnTar(ctx, f, targetDir, opt.UntarOptions)
+		pb := NewProgressReader("Unpacking ", f)
+		pb.Start()
```

Comment:
pb can be nil, so would need to wrap that in `if pb != nil` to avoid panic when STDERR is not a terminal (like when redirecting output with 2>file)